CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -184,7 +184,7 @@ public function setDefaultValues() {
     }
 
     if ($this->_refund) {
-      $defaults['total_amount'] = abs($this->_refund);
+      $defaults['total_amount'] = CRM_Utils_Money::format(abs($this->_refund), NULL, NULL, TRUE);
```

Comment:
@eileenmcnaughton @mattwire I've made the changes. It's working fine on my local.

Reply:
@monishdeb I haven't looked at this in detail but I don't think you need the abs() in there?

Reply:
I thought so but $this->_refund holds negative amount and eventually, it results to ```(50.12)``` for ```-50.12``` value, which how is the money_format(..) treat negative amount (see [here](http://php.net/manual/en/function.money-format.php))  So we need to use abs() here.

Reply:
Happy with this, I do think we need to look at refactoring that CRM_Utils_Money::format function - it's too vague with what it returns and we get wierd issues like this in different locales! See https://issues.civicrm.org/jira/browse/CRM-16460 and https://issues.civicrm.org/jira/browse/CRM-20772.  But I agree this PR is ready to merge @JoeMurray